src/main/java/com/patsi/utils/FileHelper.java

```diff
@@ -5,14 +5,16 @@
 import java.io.IOException;
 
 public class FileHelper {
-    public static File newFile(String pathname) {
-        return new File(pathname);
-    }
-    public static FileOutputStream newFileOutputStream(File f) throws IOException {
-        try{
-            return new FileOutputStream(f);
-        } catch (Exception e){
-            return null;
+    public static void newFile(String pathname, String profilePictureID, byte[] image)
```

Comment:
if this is a method for new file only, it better to check that file does not exist.
throw exception if the file already exist. 
P.s. you can throw RuntimeException if you don't wanna force catching it

Reply:
fixed